experimental/InvariantTheory/src/InvariantTheory.jl

```diff
@@ -0,0 +1,497 @@
+import Oscar.gens, AbstractAlgebra.direct_sum, Oscar.invariant_ring
+export ReductiveGroup, reductive_group, representation_matrix, group, reynolds_operator, group_ideal, canonical_representation, natural_representation
+export RepresentationReductiveGroup, representation_reductive_group, representation_on_forms, representation_matrix
+export InvariantRing, invariant_ring, fundamental_invariants, null_cone_ideal
+##########################
+#Reductive Groups
+##########################
+mutable struct ReductiveGroup
+    field::Field #characteristic zero. implement check? 
+    group::Tuple{Symbol, Int}
+    group_ideal::MPolyIdeal
+    reynolds_operator::Function
```

Comment:
This field is not used, is it? It now lives in `InvariantRing` instead, doesn't it?

Reply:
:field in InvariantRing points to ReductiveGroup.field . If it is not necessary to cache the field in ReductiveGroup (making it more of a group scheme), I could change this.

Reply:
I was actually referring to just the `reynolds_operator` field in this struct

Reply:
Sorry I misunderstood. 
Since there are two Reynolds operators in the theory for SLm: R_V and R_SLm, I have created a field for R_SLm in this data type, after a discussion with @wdecker and you. 
But R_SLm is never actually callable by the user, and this field will not be used unless the group is SLm.